Sources/BraintreePayPal/BTPayPalError.swift

```diff
@@ -27,9 +27,6 @@ enum BTPayPalError: Error, CustomNSError, LocalizedError {
     /// Unable to create BTPayPalAccountNonce
     case failedToCreateNonce
     
-    /// Unable to launch ASWebAuthenticationSession
-    case webSessionFailedToLaunch
```

Comment:
We should add that we removed this error case to the CHANGELOG

Reply:
I believe it was added in `fpti-feature` only and then removed here, so it doesn't need a CHANGELOG. @KunJeongPark can confirm

Reply:
Ah, cool - in that case... disregard 🙈